test/datetime/tokenParse.test.js

```diff
@@ -74,6 +74,17 @@ test("DateTime.fromFormat() throws if you specify meridiem with 24-hour time", (
   expect(() => DateTime.fromFormat("930PM", "Hmma")).toThrow(ConflictingSpecificationError);
 });
 
+test("DateTime.fromFormat() throws if h is used with 24-hour time and strictHours option is enabled", () => {
+  expect(() => DateTime.fromFormat("18:30", "h:mm")).not.toThrow(ConflictingSpecificationError);
```

Comment:
Please move this to a separate test, named something like 'h token accepts 24 hours when strictHours is disabled' and test that it is parsed properly instead of just testing that it does not throw.